README.md

```diff
@@ -1 +1,96 @@
-# Private_Events
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Private Events
+
+The Microverse Associations project for Ruby on Rails module that consists of creating an application named Private Events and that behaves similar to Eventbrite, which allows users to create events and then manage user signups. Users can create events, Events take place at a specific date and at a location. A user can create events. A user can attend many events. An event can be attended by many users. The main goal is to put in practice the main concepts of Associations in rails.
+
+## Built With
+- Ruby programming language
+- Rails framework
+- HTML
+- CSS
+- Bootstrap
+- The MVC design patterns.
+
+### Rails gems:
+- capybara
+- rspec-rails
+- rails-controller-testing
+- wdm
+
+### Additional gems:
+- rubocop
+
+### Documentations
+- https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations
+- https://relishapp.com/rspec/rspec-rails/v/4-0/docs
+- https://madeintandem.com/blog/setting-up-rspec-and-capybara-in-rails-5-for-testing/
+- https://www.codewithjason.com/rails-testing-hello-world-using-rspec-capybara/
+
+Readme file for the following gems:
+- https://github.com/rspec/rspec-rails/
+- https://github.com/teamcapybara/capybara
+
+
+## Authors
+### Author
+- 👤GitHub: [Jose Abel Ramirez](https://github.com/jose-Abel)
+- Linkedin: [Jose Abel Ramirez Frontany](https://www.linkedin.com/in/jose-abel-ramirez-frontany-7674a842/)
+
+### Author
+- 👤GitHub: [Guillain Bisimwa](https://github.com/guillainbisimwa)
+- Linkedin: [Guillain Bisimwa](https://www.linkedin.com/in/guillain-bisimwa-8a8b7a7b/)
+
```

Comment:
There isn't a right or wrong rule but looks more conventional to have the **Authors** section just above your **Acknowledgement** section below.

Also, replace Authors with your real names.